src/track/beatgrid.cpp

```diff
@@ -3,16 +3,22 @@
 #include <QMutexLocker>
 #include <QtDebug>
 
+#include "track/beatutils.h"
 #include "track/track.h"
 #include "util/math.h"
 
-static const int kFrameSize = 2;
+namespace {
 
 struct BeatGridData {
     double bpm;
     double firstBeat;
 };
 
+constexpr int kFrameSize = 2;
+constexpr double kBpmScaleRounding = 0.001;
```

Comment:
Previously this was 1/48 in this PR, which is a multiple of 1/12. Is there a reason for this? If so, have you considered 1/960 ?

Reply:
Anyway, this can be done into a follow-up PR if needed. I'm going to merge this now.